content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -98,19 +113,21 @@ quantities match, and haven't been changed at all, no value appears in the :guil
 column.
 
 .. image:: count_products/difference-column.png
-   :align: center
    :alt: Difference column on inventory adjustments page.
 
 At this stage, the count (:dfn:`inventory adjustment`) is recorded, but not yet applied. This means
 that the quantity on hand before the adjustment has not yet been updated to match the new, real
 counted quantity.
 
 There are two ways to apply the new inventory adjustment. The first way is to click the
-:guilabel:`Apply` button on the line at the far right of the page. The second way is to click the
-checkbox on the far left of the line. Doing so reveals new button options at the top of the page,
-one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
+:icon:`fa-save` :guilabel:`Apply` button on the line at the far right of the page. The second way is
+to click the checkbox on the far left of the line. Doing so reveals new button options at the top of
+the page, one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
 :guilabel:`Inventory Adjustment Reference / Reason` pop-up window to appear.
```

Comment:
This may have been covered previously, is does this pop-up change to "Reference" or "Reason" on a condition?

Either way, the pop-up's name is only "Inventory Adjustment"
```suggestion
:guilabel:`Inventory Adjustment` pop-up window to appear.
```